terraform/modules/hub/hub_config.tf

```diff
@@ -6,7 +6,8 @@ module "config_ecs_asg" {
   cluster             = "config"
   vpc_id              = "${aws_vpc.hub.id}"
   instance_subnets    = ["${aws_subnet.internal.*.id}"]
-  number_of_instances = "${var.number_of_availability_zones}"
+  # number_of_instances = "${var.number_of_availability_zones}"
+  number_of_instances = 0
```

Comment:
Why is this 0? Do we still need to make `config` the app exist?

Reply:
yeah we do but we're not deploying to it so until then save $